dashboard/app/models/parent_levels_child_level.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: parent_levels_child_levels
+#
+#  id              :integer          not null, primary key
+#  parent_level_id :integer          not null
+#  child_level_id  :integer          not null
+#  position        :integer
+#
+# Indexes
+#
+#  index_parent_levels_child_levels_on_child_level_id   (child_level_id)
+#  index_parent_levels_child_levels_on_parent_level_id  (parent_level_id)
+#
+
+class ParentLevelsChildLevel < ActiveRecord::Base
```

Comment:
Why is "ParentLevels" plural in this name? Doesn't one `ParentLevelsChildLevel` refer to a single parent level?

Reply:
I can't find it in the official documentation, but I'm attempting to follow the rails naming conventions outlined in various blog posts such as this one: https://gist.github.com/philsof/3ba9fb3c352c7ccf1d06